### DISCUSSION TITLE
[Camera Support]: Playback of alerts and historical camera recordings doesn't work for Reolink Duo series cameras

### DISCUSSION THREAD

**Discussion by DanGamingTV** [blakeblackshear/frigate#12549] [Camera Support]

**Comment by NickM-27** [comment]
I don't think this is really a Frigate bug, the segments aren't being pulled most likely because nginx is unable to interpret the recorded files or some other issue. We've seen it a couple times with other buggy cameras.

I have a hikvision panoramic camera that uses h265 and works without issues and in general we've not seen this issue reported widely so I'm not sure there's anything we can do. 

You might try different browsers and see if one might handle it better.

**Comment by baudneo** [comment]
Hi @DanGamingTV, sorry to bump an old topic. Did you manage to find a workaround to view duo 2 videos in frigate? I have tried FF, chrome and some other variants without luck.

**Reply by DanGamingTV** [reply]
If you're on windows, try [ungoogled-chromium](https://ungoogled-software.github.io/ungoogled-chromium-binaries/). If you're on linux, try [Thorium Browser](https://thorium.rocks/). These have worked for me for playback of the Reolink H265 footage.
If those browsers don't work, you may want to try downloading some of the H265 footage and attempting to play it in a media player on your PC - if that doesn't work then the problem may be that you don't have H265 decoders available on your system.
Hope this helps

**Reply by baudneo** [reply]
Thanks. It is h265 licensing issues. I run AMD with Linux and just assumed newer AMD iGPU platforms had HEVC HW decoding :(. Time to pull out an old 1660ti and add it to my desktop.

Here's to hoping AV1 encoding becomes a thing with onvif/rtsp.

**Comment by Gubelat** [comment]
Check this!

https://github.com/blakeblackshear/frigate/discussions/16965#discussion-8044279
